index.html

```diff
@@ -169,6 +170,7 @@ <h3>Recommendation Track specification repositories</h3>
         <li><a href="https://github.com/w3c/ttml3/pulse">TTML3</a></li>
         <li><a href="https://github.com/w3c/webvtt/pulse">WebVTT</a></li>
         <li><a href="https://github.com/w3c/adpt/pulse">ADPT</a></li>
+        <li><a href="https://github.com/w3c/tt-module-registry/pulse">TTML Extension Module Registry</a></li>
```

Comment:
I think the group decided not to move forward with this registry. I'm not sure it's a good idea to advertise it on the group's page.

Reply:
I agree, we should not add it at this time.

Reply:
removed